src/components/Menu/index.jsx

```diff
@@ -1,14 +1,31 @@
+import { useContext } from "react";
 import MenuItem from "../MenuItem";
+import ThemeSwitcher from "../ThemeSwitcher";
 import styles from "./Menu.module.css";
+import { ThemeContext } from "../../context/ThemeContext";
 
 const Menu = ({ homeRef, aboutRef, contactRef, skillsRef, projectsRef }) => {
+  const { darkMode, switchTheme } = useContext(ThemeContext);
+
+  const menuItems = [
```

Comment:
Make this menuItems to passable as props

Reply:
Passed the menuItems as props from Header component.